internal/eval/eval_test.go

```diff
@@ -38,42 +38,43 @@ func setupTest(store *dstore.Store) *dstore.Store {
 func TestEval(t *testing.T) {
 	store := dstore.NewStore(nil)
 
-	testEvalMSET(t, store)
-	testEvalPING(t, store)
-	testEvalHELLO(t, store)
-	testEvalSET(t, store)
-	testEvalGET(t, store)
-	testEvalDebug(t, store)
-	testEvalJSONARRPOP(t, store)
-	testEvalJSONARRLEN(t, store)
-	testEvalJSONDEL(t, store)
-	testEvalJSONFORGET(t, store)
-	testEvalJSONCLEAR(t, store)
-	testEvalJSONTYPE(t, store)
-	testEvalJSONGET(t, store)
-	testEvalJSONSET(t, store)
-	testEvalJSONTOGGLE(t, store)
-	testEvalJSONARRAPPEND(t, store)
-	testEvalTTL(t, store)
-	testEvalDel(t, store)
-	testEvalPersist(t, store)
-	testEvalEXPIRE(t, store)
-	testEvalEXPIRETIME(t, store)
-	testEvalEXPIREAT(t, store)
-	testEvalDbsize(t, store)
-	testEvalGETSET(t, store)
-	testEvalHSET(t, store)
-	testEvalPFADD(t, store)
-	testEvalPFCOUNT(t, store)
-	testEvalHGET(t, store)
-	testEvalPFMERGE(t, store)
-	testEvalJSONSTRLEN(t, store)
-	testEvalJSONOBJLEN(t, store)
-	testEvalHLEN(t, store)
-	testEvalSELECT(t, store)
-	testEvalLLEN(t, store)
-	testEvalGETEX(t, store)
-	testEvalJSONNUMINCRBY(t, store)
+	// testEvalMSET(t, store)
```

Comment:
@psrvere can you please uncomment these.

Reply:
Apologies. Corrected.